lib/godmin/authorization/policy_finder.rb

```diff
@@ -17,10 +17,20 @@ def find(object)
             end
 
           if Godmin.namespace
-            "#{Godmin.namespace.classify}::#{klass}Policy"
-          else
-            "#{klass}Policy"
+            prefix = "#{Godmin.namespace.classify}::"
           end
+          policy_class = "#{klass}Policy"
+          add_prefix_once(prefix, policy_class)
+        end
+
+        private
+
+        def add_prefix_once(prefix, s)
```

Comment:
* is "random_duck_typed_object_that_can_be_interpolated" descriptive enough?
* interpolatable?